### PR TITLE
Add TypeScript definitions

### PR DESCRIPTION
**Pull request by VVatashi**
This can make it easier to use this library from TypeScript projects

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module 'tripcode' {
+  export default function tripcode(key: string): string;
```
Comment (KenanY) [inline types.d.ts:2]: Double checking: is `default` correct here if we're using `module.exports =`? I thought we might need something like:
```typescript
declare function tripcode(key: string): string;
export = tripcode;
```

**Review by KenanY — APPROVED** [review]
Just one small question but otherwise LGTM.

File: types.d.ts (hunk 1 above)
Comment (VVatashi) [reply types.d.ts:2]: Yes, you are right. I have re-read the documentation on .d.ts, and it says to use the old `export =` syntax for better compatibility. Although in modern versions of the TS both syntaxes seem to work the same

**Comment by KenanY** [comment]
Landed as 66b0b871d1a4d67cd7488a642292c772965ae802 and published in v1.4.0. Thank you!

**Pull request closed without merging**